yarn.lock

```diff
@@ -2994,15 +2994,14 @@
   dependencies:
     "@sinonjs/commons" "^1.7.0"
 
-"@slorber/static-site-generator-webpack-plugin@^4.0.4":
-  version "4.0.4"
-  resolved "https://registry.yarnpkg.com/@slorber/static-site-generator-webpack-plugin/-/static-site-generator-webpack-plugin-4.0.4.tgz#2bf4a2545e027830d2aa5eb950437c26a289b0f1"
-  integrity sha512-FvMavoWEIePps6/JwGCOLYKCRhuwIHhMtmbKpBFgzNkxwpa/569LfTkrbRk1m1I3n+ezJK4on9E1A6cjuZmD9g==
+"@slorber/static-site-generator-webpack-plugin@^4.0.6":
+  version "4.0.6"
+  resolved "https://registry.yarnpkg.com/@slorber/static-site-generator-webpack-plugin/-/static-site-generator-webpack-plugin-4.0.6.tgz#fd460d449800e1ea6e917bf683683db8c8a230bb"
+  integrity sha512-p1EcMCK4Y/y6+O0kAa1HJu7wxx6mHJcOv6z6nFi35CAnO5Krz0fBgfDUAJnmN2Xh1pw/+LCDsN9paKQ5Hou7zA==
   dependencies:
-    bluebird "^3.7.1"
-    cheerio "^0.22.0"
-    eval "^0.1.8"
-    webpack-sources "^1.4.3"
+    eval "^0.1.4"
```

Comment:
This range change looks a bit weird, though

Reply:
yes good catch, bad merge, will publish another version